### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by TBMSP** [xspanger3770/GlobalQuake#147] [Ideas]
I have reviewed the code and made some modifications for my own use and testing, but I have realized that several of them can be implemented well in the future:

- "Custom GeoJson": Add support for adding or removing geojson.
- "Custom GeoJson": With json files, in some of them, the "Name" parameter does not exist, so if a json importer is added in the future, it should also be possible to specify some parameters.
- Change the sounds.
- Option to function as a console without UI.
- "Custom GeoJson": In Regions, it doesn't detect some regions correctly when the epicenter is on the coast, and I think the "In the Middle of Nowhere" message could be improved.
- Option to export/import events or databases for later review.
- Option to export event map images or plots.
- Documentation.
- In the station selection it is fine, but I would also like a list with the networks and station codes as well as a search bar.

I hope to collaborate with the project in the future once I finish researching it thoroughly. Excellent work, greetings!

**Comment by YacineBoussoufa** [comment]
- I agree for the geojson, it uses a fixed template of a repository and it's not standardized, and therefore you can't add custom files
- The sounds are work in progress #96
- "In the Middle of Nowhere" will be fixed as we are working implementing FE Regions
- Event maps can be exported if the "Enable Earthquake Reports", located inthe settings' Debug tab, is enabled
